viewing_party/party.py

```diff
@@ -1,23 +1,160 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if not title or  not genre or not rating:
+        return None
+    else:
+        movies = {}
+        movies["title"] = title
+        movies["genre"] = genre
+        movies["rating"] = rating
+        return movies 
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data,movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    if title in user_data["watchlist"]:
+        user_data["watchlist"].remove(title)
+        user_data["watched"].append(title)
+        return user_data
+    return user_data

```

Comment:
I think this function isn't working as you expect. 
- `user_data` is a dictionary where the value of each key-value pair is a list. 
- The lists inside user data, such as `user_data["watchlist"]`, are lists where each element is a dictionary of data which represents a movie.  
- The `title` parameter is a string. 

If we try to look for the string `title` in `user_data["watchlist"]`, this will always fail since `user_data["watchlist"]` contains dictionaries.

How could we rewrite this function to look inside each movie dictionary in `user_data["watchlist"]` and check if the title of the movie matches the `title` parameter?